mllib/src/main/scala/org/apache/spark/ml/feature/RFormula.scala

```diff
@@ -78,13 +78,20 @@ class RFormula(override val uid: String) extends Estimator[RFormulaModel] with R
   /** @group getParam */
   def getFormula: String = $(formula)
 
+  /** Whether the formula specifies fitting an intercept. */
+  def hasIntercept: Boolean = {
```

Comment:
Could this method be package private? We don't expose parsed terms to users. `intercept` should be treated the same.